### PR TITLE
FEAT: `radians` and `deg2rad` ufunc implementation

### PR DESCRIPTION
**Pull request by SwayamInSync**
## Copilot Summary
This pull request adds support for converting degrees to radians for quad-precision types by implementing the `radians` and `deg2rad` universal functions (ufuncs) in the codebase. It includes both the implementation of these functions for quad and long double types, their registration with NumPy, updates to the release tracker, and comprehensive tests to ensure correctness.

### New ufunc implementations and registration

* Added `quad_radians` and `ld_radians` functions to compute radians from degrees for quad-precision (`Sleef_quad`) and long double types, respectively, in `ops.hpp`. [[1]](diffhunk://#diff-ad4140fa03d374f246bdc4d441d868fa759788f2d27a7a29ca259b16780f0850R246-R254) [[2]](diffhunk://#diff-ad4140fa03d374f246bdc4d441d868fa759788f2d27a7a29ca259b16780f0850R458-R467)
* Registered the new `radians` and `deg2rad` ufuncs in the NumPy interface initialization, making them available as universal functions for quad-precision arrays.

### Documentation and tracking

* Updated `release_tracker.md` to mark `radians` and `deg2rad` as implemented and tested for quad-precision.

### Testing

* Added thorough parameterized tests for both `radians` and `deg2rad` ufuncs, covering a variety of input cases including standard angles, negative values, infinities, NaNs, and edge cases.

### CODE REVIEW

File: quaddtype/numpy_quaddtype/src/ops.hpp [hunk 1]
```diff
@@ -243,6 +243,15 @@ quad_atanh(const Sleef_quad *op)
     return Sleef_atanhq1_u10(*op);
 }
 
+static inline Sleef_quad
+quad_radians(const Sleef_quad *op)
+{
+    // radians = degrees * π / 180
+    static const Sleef_quad one_eighty = Sleef_strtoq("180.0", NULL);
```
[the remark this answers is not in the document]
Comment (SwayamInSync) [reply quaddtype/numpy_quaddtype/src/ops.hpp]: Easy and accurate, sometimes conversion from int or float gives precision errors (although here it shouldn't technically)

We can also go with qutil, but it seems overkill

**Comment by SwayamInSync** [comment]
Taking this in!
